ui/pages/confirmation/confirmation.js

```diff
@@ -91,10 +91,12 @@ const alertStateReducer = produce((state, action) => {
  *
  * @param {object} pendingConfirmation - a pending confirmation waiting for
  * user approval
+ * @param {object} state - The state object consist of required info to determine alerts.
+ * @param state.unapprovedTxsCount
  * @returns {[alertState: object, dismissAlert: Function]} A tuple with
  * the current alert state and function to dismiss an alert by id
  */
-function useAlertState(pendingConfirmation) {
+function useAlertState(pendingConfirmation, { unapprovedTxsCount } = {}) {
```

Comment:
Destructuring object here intentionally because we don't want to pass entire object to `useEffect` in order to avoid React warnings.